### DISCUSSION TITLE
Does the visualiser support "Multiple Targets" for parallel states?

### DISCUSSION THREAD

**Discussion by earslap** [statelyai/xstate#4424] [Q&A] [answered]
Does not look like it. Wondering if this is a bug or if this is unsupported. Importing this machine: https://xstate.js.org/docs/guides/transitions.html#multiple-targets in the visualiser removes the array for the multiple targets under DEACTIVATE.

**Comment by davidkpiano** [comment, marked as answer]
Not yet. I have a PR in the works for this.
